bundles/org.openhab.binding.broadlinkthermostat/NOTICE

```diff
@@ -0,0 +1,13 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
```

Comment:
Can you add license information about the used library here?

Reply:
It is already there in the section "Third-party Content". Anything else that is missing or was it just overlooked?

Reply:
Added Third Party License section.